scripter/lua/lua.go

```diff
@@ -70,6 +70,39 @@ func (l *luaScripter) SetGlobalFn(name string, fn func() string) error {
 	return l.SetStringFunction(name, fn)
 }
 
+// Check whether scripter can handle incoming connection
+// Get all scripts for a given service and pass the string to each script
+func (l *luaScripter) CanHandle(message string) (bool, error) {
+	// TODO: Is checking all of them the way to go?
+	for _, ls := range l.scripts[l.name] {
+		result, err := canHandleScript(ls, message)
```

Comment:
Print error, but continue, if a script errors, it shouldn't stop, just say that it doesn't work.